guides/user/plugins/index.md

```diff
@@ -200,6 +202,33 @@ spinnaker:
 
 Note: `hal plugins enable` and `hal plugins disable` enable or disable _all_ plugins, so use with caution.
 
+### Plugin configuration without Halyard
+
+To avoid each service restarting and downloading the plugin, _do not_ add the plugin using Halyard. Instead, configure the plugin in the service's local file. For example, if your plugin extends Orca, add configuration to your `orca-local.yml` file.
```

Comment:
In the above section, you have the download coming before the restart. If that's how it happens, maybe keep that same order here.

Reply:
Thanks for catching that! Download happens after restart.